listener/avalon_listener/tcs_work_order_handler.py

```diff
@@ -55,7 +56,8 @@ def __init__(self, kv_helper, max_wo_count):
         self.workorder_count = 0
         self.max_workorder_count = max_wo_count
         self.workorder_list = []
-
+        self.INVALID_WORKLOAD_MESSAGE = \
+            'CreateWorkloadProcessor function returned null'
```

Comment:
This doesn't look good. listener ideally shouldn't check for exact error message string of enclave manager. Instead use an error code.

Reply:
This is what we get it from enclave manager. value error and message as CreateWorkloadProcessor function returned null

Reply:
Listener need to check only for error code enclave manager. It should not compare the error message from enclave manager. If you need specific error code then we need to modify enclave manager to return the specific error code